docs/cloud-deploy/README.md

```diff
@@ -0,0 +1,381 @@
+# **Deploying Mailchimp Open Commerce on to a Cloud Hosted Kubernetes Cluster**
+
+## **Introduction**
+
+MailChimp Open Commerce is an open source, API-first, modular commerce
+stack made for technical, growth-minded retailers.
+
+The purpose of this document is to provide guidance on deploying an Open
+Commerce environment on to a cloud hosted Kubernetes cluster using the
+provided Helm charts to streamline the deployment process. This guide
+has been written to be as cloud agnostic as possible so some reference
+to your chosen cloud providers documentation may be required for
+particulars (such as SSL certificate generation and hosting, load
+balancing/ingress, etc...) although the basic steps and deployment
+approach (Helm) should be the same regardless of chosen cloud.
+
+## **Prerequisites**
+
+As this document is focused upon deploying MailChimp Open Commerce,
+using Helm, some working knowledge of your chosen cloud provider and
+Kubernetes is required. For example, you will need a cloud hosted
+Kubernetes environment, and relevant administration/deployment
+permissions to said environment, before being able to proceed with the
+below. You will also need **kubectl** installed and configured to
+proceed.
+
+This documentation has been written for and tested on Linux specifically
+(Fedora 35), but all the included tools and components are also
+available for other operating systems such as Windows and Mac. Please
+consult each components/tools bespoke documentation for guidance
+regarding non-Linux installation and configuration.
+
+## **Components**
+
+Cloud Provider: Cloud vendors offer a multitude of third-party hosting
+services and solutions, including compute, networking, and
+storage. As mentioned, an account with a chosen cloud provider is
+required, with sufficient access privileges, to follow along with this
+guide.
+
+[Kubernetes](https://kubernetes.io/): Kubernetes, also known as K8s,
+is an open-source system for automating deployment, scaling, and
+management of containerized applications. You will require an active
+cloud hosted Kubernetes environment before being able to proceed with
+the following.
+
+[Kubectl](https://kubernetes.io/docs/tasks/tools/): The Kubernetes
+command-line tool, kubectl, allows you to run commands against
+Kubernetes clusters. You can use kubectl to deploy applications, inspect
+and manage cluster resources, and view logs.
+
+[Kubernetes Ingress
+Controller](https://kubernetes.io/docs/concepts/services-networking/ingress/):
```

Comment:
```suggestion
[Kubernetes Controller](https://kubernetes.io/docs/concepts/services-networking/ingress/):
```

Reply:
Unclear why we break these up over two lines. Renders find but makes the markdown harder to read